openedx/core/djangoapps/course_apps/rest_api/v1/views.py

```diff
@@ -139,7 +140,21 @@ def get(self, request: Request, course_id: str):
                 "request": request,
             }
         )
-        return Response(serializer.data)
+        course_apps = serializer.data
+        unsortable_apps = []
+        sort_as = settings.COURSE_APPS_ORDER
+        for app in serializer.data:
+            try:
+                sort_index = sort_as.index(app['id'])
+            except ValueError:
+                sort_index = None
+
+            if sort_index is not None:
+                course_apps[sort_index] = app
+            else:
+                unsortable_apps.append(app)
+        course_apps = course_apps + unsortable_apps
```

Comment:
I think this code can be greatly simplified if `COURSE_APPS_ORDER` is a dict mapping the app id to priority. 

Then this can be reduced to: 
```
sorted(COURSE_APPS_ORDER, key=lambda x: COURSE_APPS_ORDER.get(x, float('inf'))
```

Reply:
Additionally, I feel like this is a presentation-related change, so the sorting code should probably be in the frontend right?